.github/workflows/check_code_style.yml

```
@@ -0,0 +1,21 @@
+name: Check Code Style
+
+on:
+  push:
+    branches:
+      - "**"
+
+jobs:
+  check_code_style:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Initialize and update submodule
+        run: git submodule update --init --recursive
```

Instead should we do:
```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: 'recursive'
```